cors/redirect-preflight.htm

```diff
@@ -12,54 +13,62 @@
 <div id=log></div>
 <script>
 
-var req_c = 0 // Request count for cache busting and easy identifying of request in traffic analyzer
+// Request count for cache busting and easy identifying of request in traffic
+// analyzer.
+var req_c = 0;
+
+var CROSSDOMAIN_URL = CROSSDOMAIN + 'resources/cors-makeheader.py?';
 
 /*
- * Redirection with preflights
+ * Redirection with preflights.
  */
-
 function redir_preflight(code) {
     test(function() {
-        var client = new XMLHttpRequest(),
-            redirect = CROSSDOMAIN + 'resources/cors-makeheader.py?headers=x-test&' + req_c++
+        var client = new XMLHttpRequest();
+        var redirect =
+          encodeURIComponent(CROSSDOMAIN_URL + 'headers=x-test&' + req_c++);
 
-        client.open('GET', CROSSDOMAIN + 'resources/cors-makeheader.py?'
-                           + 'headers=x-test&location=' + encodeURIComponent(redirect)
-                           + '&code=' + code + '&preflight=' + code + '&' + req_c++,
-                    false)
-        client.setRequestHeader('x-test', 'test')
+        client.open('GET',
+                    CROSSDOMAIN_URL + 'headers=x-test&location=' + redirect
+                                    + '&code=' + code + '&preflight=' + code
+                                    + '&' + req_c++,
+                    false);
+        client.setRequestHeader('x-test', 'test');
         assert_throws(null, function() { client.send(null) });
 
-    },
-    'Redirect ' + code + ' on preflight')
+    }, 'Redirect ' + code + ' on preflight');
```

Comment:
Yes.

The spec says in steps 6 and 7 of [CORS preflight fetch](https://fetch.spec.whatwg.org/#cors-preflight-fetch):

> (6) If a CORS check for request and response returns success and response’s status is an ok status, run these substeps [...]. (7) Otherwise, return a network error. 

There is no explicit mention about 3XX redirection status codes. So maybe this is not the best place for this test? Is this what you are suggesting with your comment?